src/index.ts

```diff
@@ -2667,10 +2667,19 @@ function FlatpickrInstance(
 
 /* istanbul ignore next */
 function _flatpickr(
-  nodeList: NodeList | HTMLElement[],
+  nodeList: ArrayLike<Node>,
   config?: Options
 ): Instance | Instance[] {
-  const nodes: HTMLElement[] = Array.prototype.slice.call(nodeList); // static list
+  // spare ourselves some cycles & errors
```

Comment:
The check is unnecessary. If a user decides to pass in `null`, let them handle the error instead of accommodating this unreasonable behavior.